Add simplified Chinese translation

virtualwiz opened the pull request:
A wonderful introduction to the applications of FT, I really really like it. Would recommend this artwork to my university professors so more students can get better understanding, thanks so much!

A suitable language label can be 简体中文
Would recommend a title for the article 傅里叶变换交互式入门

Jezzamonn commented on content/content-zh-cn.md:
```diff
@@ -0,0 +1,449 @@
+
+最明显的例子就是声音 —— 当我们听到声音时，我们听不到那条波浪线，但我们听到构成声音的正弦波的不同频率。
+
+<button id="together-button" class="button">Play Full Wave</button>
```
So there are some buttons that also need to be translated, I'll leave comments on the lines that still have some English text.

Here it's "Play Full Wave"

virtualwiz replied:
Wow, sorry for that, I will make an update

Jezzamonn commented on content/content-zh-cn.md:
```diff
@@ -0,0 +1,449 @@
+
+<button id="together-button" class="button">Play Full Wave</button>
+
+<button id="split-button-1" class="button">Play High Frequency</button>
```
"Play High Frequency"

Jezzamonn commented on content/content-zh-cn.md:
```diff
@@ -0,0 +1,449 @@
+
+<button id="split-button-1" class="button">Play High Frequency</button>
+
+<button id="split-button-2" class="button">Play Low Frequency</button>
```
Play Low Frequency

Jezzamonn commented on content/content-zh-cn.md:
```diff
@@ -0,0 +1,449 @@
+<canvas id="square-wave-build-up" class="sketch" width=500 height=500></canvas>
+<input id="square-wave-build-up-slider" type="range" min="0" max="1" value="0" step="any" >
+
+<button id="square-wave-button" class="button">Play Wave</button>
```
Play Wave

Jezzamonn commented on content/content-zh-cn.md:
```diff
@@ -0,0 +1,449 @@
+<canvas id="wave-draw-split" class="sketch" width=500 height=500></canvas>
+</div>
+<input id="wave-draw-slider" type="range" min="0" max="1" value="1" step="any">
+<button id="wave-draw-button" class="button">Play Wave</button>
```
Play Wave

Jezzamonn commented on content/content-zh-cn.md:
```diff
@@ -0,0 +1,449 @@
+<div class="multi-container">
+<div class="sketch" >
+    <canvas id="wave-draw" class="sketch-child" width=500 height=300></canvas>
+    <p id="wave-draw-instruction" class="instruction wave-instruction">Draw here!</p>
```
Draw here!

Jezzamonn commented on content/content-zh-cn.md:
```diff
@@ -0,0 +1,449 @@
+<div class="sketch" >
+    <canvas id="draw-zone" class="sketch-child" width=500 height=500></canvas>
+    <p id="draw-zone-instruction" class="instruction">Draw here!</p>
+    <button id="draw-zone-undo-button" class="button embedded-button">Undo</button>
```
Undo

Jezzamonn commented on content/content-zh-cn.md:
```diff
@@ -0,0 +1,449 @@
+要了解更多信息，你可以看看这些非常好的资源：
+
+[An Interactive Guide To The Fourier Transform](https://betterexplained.com/articles/an-interactive-guide-to-the-fourier-transform/)  
+A great article that digs more into the mathematics of what happens.
```
FYI, this text is still in English. The part in the [] should stay the same because that's the title, but you might want to update the description. Also possibly worth mentioning it's in English.

virtualwiz commented:
Thanks for reminding, added these translations. I also added another Wikipedia entry for the corresponding Chinese Fourier transform Wikipedia page.
Cheers!

Jezzamonn commented:
Looks good! Thanks for the translation!

Jezzamonn commented:
One other follow up on this: At the top of the page I include a "translated by ____" section. Does this work:

由virtualwiz翻译 - with a link to your GitHub like in the section at the end? Normally I'd use your full name but if you'd rather be credited as virtualwiz that is ok too :)

virtualwiz commented:
> One other follow up on this: At the top of the page I include a "translated by ____" section. Does this work:
> 
> 由virtualwiz翻译 - with a link to your GitHub like in the section at the end? Normally I'd use your full name but if you'd rather be credited as virtualwiz that is ok too :)

Thanks a lot, it reads great :)

virtualwiz commented:
> One other follow up on this: At the top of the page I include a "translated by ____" section. Does this work:
> 
> 由virtualwiz翻译 - with a link to your GitHub like in the section at the end? Normally I'd use your full name but if you'd rather be credited as virtualwiz that is ok too :)

My full name is 杜尚明，Writing like 由 杜尚明 翻译 feels more normal, thanks a lot :)

Jezzamonn commented:
Alright, just updated! Should be there once the web cache resets!

virtualwiz commented:
Thanks a lot!
I noticed there is an error 404 when tried to access the translated page, is this caused by web cache not refreshed or some other issue, please?
![zhcn-404-issue](https://user-images.githubusercontent.com/14255046/57217055-7b683200-6fe9-11e9-8ffd-9135a82bca5d.png)

Jezzamonn commented:
Ah, ok. I made some changes to the capitalisation but it looks like the way
the webpage and my computer handle capital/lowercase files is a bit weird
and made me muck that up.

The page is at http://www.jezzamon.com/fourier/zh-cn.html, but the link was
pointing to one with different capitalisation. I've updated the page to
point to the correct one now (one cache resets it'll be up)

On Mon, May 6, 2019 at 2:27 AM Matthew S Du <notifications@github.com>
wrote:

> Thanks a lot!
> I noticed there is an error 404 when tried to access the translated page,
> is this caused by web cache not refreshed or some other issue, please?
> [image: zhcn-404-issue]
> <https://user-images.githubusercontent.com/14255046/57217055-7b683200-6fe9-11e9-8ffd-9135a82bca5d.png>
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Jezzamonn/fourier/pull/9#issuecomment-489560363>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ACOXODSQ5HJ4Y6XMFBPSIZTPT72X5ANCNFSM4HHKOZ6Q>
> .
>

virtualwiz commented:
Now it works perfectly, thanks! :champagne:
